src/pages/Accounts/AddAccountModal.tsx

```diff
@@ -91,14 +92,18 @@ export const AddAccountModal = () => {
 
   const menuOptions = useMemo(() => {
     const chainAdapterManager = getChainAdapterManager()
-    return chainIds.map(chainId => {
-      const assetId = chainAdapterManager.get(chainId)?.getFeeAssetId()
-      const asset = assetId ? assets[assetId] : undefined
-      if (!asset) return null
-      const key = chainId
-      const chainOptionsProps = { chainId, setSelectedChainId, asset, key }
-      return <ChainOption {...chainOptionsProps} />
-    })
+    return chainIds
+      .map(chainId => {
```

Comment:
could reduce of map + filter

Reply:
Ty sir, as this is ready to go in I won't touch in this PR.